src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -282,6 +282,7 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
                   'accent-text': groupNameLength,
                   enabled: groupNameLength && !nameError.length,
```

Comment:
This is actually still needed to give the enabled label correct style

Reply:
Since this condition and https://github.com/wireapp/wire-webapp/pull/13261/files#diff-374d17b2a9898c44a7be4d7d1e40a329f71a2d6b53c865e052296e2e4988646aR285 are the exact negation of one-another, let's extract this to a variable earlier in the file ;)